tools/topology/topology2/cavs-mixin-mixout-efx-hda.conf

```diff
@@ -137,4 +138,8 @@ Object.Base.route [
 		source 'copier.HDA.4.1'
 		sink 'copier.host.3.1'
 	}
+	{
+		source 'host-copier.0.playback'
+		sink 'gain.1.1'
+	}
```

Comment:
Can we pass `pcm_id` to host-copier-gain-mixin-playback.conf, and set route there?

Reply:
@bardliao thats what I first considered. But moving the route from/to the host-copier makes it easier for me to passthrough pipelines later on.